recipes/wurst/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "wurst" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ddb56c990945a73e42af3773c712e6ca144244d4584b144d0777a309ffe4e40d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - wurst-cli = wurst.bin.wurst_cli:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - setuptools
+  run:
+    - appdirs
+    - constructive_geometries
+    - docopt
+    - numpy
+    - pandas
+    - python >=3.4
+    - python-json-logger
+    - toolz
+    - tqdm
+    - wrapt
+
+test:
+  imports:
+    - wurst
+  requires:
+    - pip
+  commands:
+    - pip check
+    - wurst-cli --help
+
+about:
+  home: https://github.com/polca/wurst
+  summary: 'Wurst is a python package for linking and modifying industrial ecology models'
+  license: BSD-3-Clause
```

The file is formatted weird, but it's actually:
```suggestion
  license: BSD-2-Clause
```